references/segmentation/datasets.py

```diff
@@ -0,0 +1,178 @@
+import torch
+import torchvision
+from PIL import Image
+from torchvision import transforms as T
+
+import os
+import numpy as np
+import random
+
+from pycocotools import mask as coco_mask
+
+
+class VOC(torchvision.datasets.VOCSegmentation):
+    def __init__(self, root, image_set='train', transforms=None):
+        super(VOC, self).__init__(root, image_set=image_set)
+        self.transforms = transforms
+
+    def __getitem__(self, idx):
+        image, target = super(VOC, self).__getitem__(idx)
+        if self.transforms is not None:
+            image, target = self.transforms(image, target)
+        return image, target
+
+    @property
+    def num_classes(self):
+        return 21
+
+class SBDD(object):
+    def __init__(self, root, image_set='train', transforms=None):
+        self.root = root
```

Comment:
os.path.expanduser(root)

Reply:
I'll definitely improve those datasets.

BTW, there has been a SBD dataset that has been recently merged into master, so I'll be inheriting from it instead most probably